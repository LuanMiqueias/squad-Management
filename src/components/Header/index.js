***REMOVED***
import styles from "./style.module.css";

const Header = () => {
***REMOVED***
    <header className={`${styles.header} container`}>
      <div className={`${styles.headerContent} content`}>
        <div className={styles.logo}>Squad Management Tool</div>
        <div className={styles.navContainer}>
          <nav>
            <a href="/#">
              John Smith <span className="span-img-fake">JS</span>
            </a>
          </nav>
***REMOVED***
      </div>
    </header>
***REMOVED***
***REMOVED***
export default Header;
