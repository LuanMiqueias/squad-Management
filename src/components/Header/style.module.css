.header{
  width: 100%;
  height: 50px;
  background: var(--gradient-background);
  display: flex;
}
.headerContent{
  display: flex;
  flex: 1;
  align-items: center;
  justify-content: space-between;
  color:var(--white)

}

.navContainer nav a{
  display: flex;
  align-items: center;
  gap:8px;
  font-size: .88rem;
  font-weight: 400;
}
