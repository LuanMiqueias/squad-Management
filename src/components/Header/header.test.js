import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Header from "./";

it("should match snapshot", () => {
  const { container } = render(
    <BrowserRouter>
      <Header />
    </BrowserRouter>
  );

  expect(container.firstChild).toMatchSnapshot();
});
