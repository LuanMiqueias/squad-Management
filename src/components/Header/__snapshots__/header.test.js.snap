// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<header
  class="header container"
>
  <div
    class="headerContent content"
  >
    <a
      class="logo"
      href="/"
    >
      Squad Management Tool
    </a>
    <div
      class="navContainer"
    >
      <nav>
        <a
          href="/#"
        >
          John Smith 
          <span
            class="span-img-fake"
          >
            JS
          </span>
        </a>
      </nav>
    </div>
  </div>
</header>
`;
